### ISSUE TITLE
Puppetboard on PE complains about PuppetDB version

### ISSUE THREAD

**Issue by tuxmea** [voxpupuli/puppetboard#711]
**Describe the bug**

Running PuppetBoard 4.0.3 on a PE platform causes the puppetboard container to never properly start:

```
[2022-09-11 10:33:02 +0000] [24] [INFO] Booting worker with pid: 24
INFO:puppetboard.utils:PuppetDB version: 7.11.1-20220809_222149-g0b0b67c
ERROR:puppetboard.utils:The minimum supported version of PuppetDB is 5.2.0
[2022-09-11 10:33:13 +0000] [24] [INFO] Worker exiting (pid: 24)
[2022-09-11 10:33:13 +0000] [25] [INFO] Booting worker with pid: 25
INFO:puppetboard.utils:PuppetDB version: 7.11.1-20220809_222149-g0b0b67c
ERROR:puppetboard.utils:The minimum supported version of PuppetDB is 5.2.0
[2022-09-11 10:33:43 +0000] [25] [INFO] Worker exiting (pid: 25)
```

**Puppetboard version**

4.0.3

**Environment and installation method**

* Centos 7 and Docker version 20.10.18, build b40c2f6
* Using a profile:
```
  $tag_name = '3.6.1'  # <--- working
  # $tag_name = '4.0.3' # <--- not working

  include docker

  docker::image { 'ghcr.io/voxpupuli/puppetboard':
    image_tag => $tag_name,
  }

  docker::run { 'puppetboard':
    image => "ghcr.io/voxpupuli/puppetboard:${tag_name}",
    env   => [
      'PUPPETDB_HOST=127.0.0.1',
      'PUPPETDB_PORT=8080',
      "PUPPETBOARD_PORT=${port}",
    ],
    net   => 'host',
  }

```

**Comment by gdubicki** [comment]
Hi @tuxmea! Thanks for reporting.

I was just hacking something Puppetboard-related so I can fix it immediately. :)

**Comment by gdubicki** [comment]
Fixed in [v4.0.5](https://github.com/voxpupuli/puppetboard/releases/tag/v4.0.5).
